sympy/logic/tests/test_boolalg.py

```diff
@@ -843,3 +844,31 @@ def test_relational_simplification():
     assert Or(Eq(x,y), x >= 1, 2 < y, y >= 5, z < y).simplify() == (Eq(x, y) | (x >= 1) | (y > Min(2, z)))
     assert And(Eq(x,y), x >= 1, 2 < y, y >= 5, z < y).simplify() == (Eq(x, y) & (x >= 1) & (y >= 5) & (y > z))
     assert (Eq(x, y) & Eq(d, e) & (x >= y) & (d >= e)).simplify() == (Eq(x, y) & Eq(d, e) & (d >= e))
+
+
+def test_relational_simplification_numerically():
+    def test_simplification_numerically_function(original, simplified):
+        symb = original.free_symbols
+        n = len(symb)
+        valuelist = list(set(list(combinations(list(range(-(n-1),n))*n, n))))
+        for values in valuelist:
+            sublist = dict(zip(symb, values))
+            originalvalue = original.subs(sublist)
+            simplifiedvalue = simplified.subs(sublist)
+            if originalvalue != simplifiedvalue:
+                # In case something fails, show clearly which case
+                print("Original: {}\nand simplified: {}\ndo not evaluate to the same value for {}".format(original, simplified, sublist))
+            assert originalvalue == simplifiedvalue
```

Comment:
You can add this to the end of the assert, like `assert originalvalue == simplifiedvalue, "Original: {}\nand simplified: {}\ndo not evaluate to the same value for {}".format(original, simplified, sublist)`. That will make it the error message of the AssertionError.

Reply:
Thanks!